### DISCUSSION TITLE
Meshcommander don't connect

### DISCUSSION THREAD

**Discussion by Veryslowraptor** [Ylianst/MeshCentral#6446] [Issues]
Hello! Recently i bought an EliteDesk 800 g3 mini pc wiht intel vPro capabilities. I have activated the AMT and I tried to connect. First, the mashcommander connects, i can power up and down the EliteDesk remotely but doesn't connect to KVM. Next I have conected a monitor (old tv) via displayport to hdmi cable and now i can connect to kvm but only when EliteDesk is turned off. Than I turn it on and all works but, iff the EliteDesk is already on, i can not connect.

**Comment by si458** [comment]
Are u sure you are using meshcommander?
As this doesn't seem right?

If u can connect and use meshcommander locally on the device,
Then u should be able to use meshcommander on another device using the first devices ip address as the host, and control it while the device is switched off

**Comment by Veryslowraptor** [comment]
I didn't use meshcommander locally on the same computer... I have used it
on a laptop. My EliteDesk mini has AMT capability and I have conected from
an Asus laptop all the time... From the start

**Comment by si458** [comment]
Seems weird u can't connect to the amt when the device is turned off?

Have u checked ur router etc to make sure the device gets an IP address when it's powered off?

Can u ping it? Can u view the amt web panel? `http://ipaddress:16992` `https://ipaddress:16993`

**Comment by Veryslowraptor** [comment]
I can connect only when is turned off and then I power up and remote
control works bun if i am disconnecting i can not reconnect. I need to
power down the EliteDesk to be able connect again.

**Reply by si458** [reply]
This to me sounds like a computer issue? Something is wrong with the amt maybe
Have u tried factory resetting the amt by
Removing the bios battery and power plug and holding down the power button?
Then going back into the intel amt bios and setting like a different password and activating it again?

**Comment by Veryslowraptor** [comment]
First, before i have connected the monitor, i could connect every time even
when EliteDesk was powered on or of but remote control didn't work. I have
connected the monitor and now i can connect only when EliteDesk is powered
off, but now remote control is working. (Because i am connecting to remote
control when EliteDesk is powered off and after i am powering up and
remains connected. But if is disconnecting, remains disconnected and I can
not reconnect

**Comment by Veryslowraptor** [comment]
No, i didn't do that, but I will. Thanks for trying to help 🙂
